source/chapters/cookbook.rst

```diff
@@ -0,0 +1,55 @@
+.. include:: /shortcuts.rstext
+
+.. _cookbook:
+
+Cookbook (Examples)
+*******************
+
+The Cookbook is here to give Mixxx users examples to work off of. The recipes both cover common or interesting use cases, and progressively explain more complex detail. This chapter moves beyond simple introductory examples and demonstrates concepts that will help users create awesome music, as well as the approach to achieving this.
+Each recipe will focus on one specific aspect of Mixxx, combining different features in interesting ways to fully explore its niche.
+
+Deck Cloning
+============
+
+When deck cloning (also called "instant doubles" in some other DJ software) is activated, the currently loaded song, playback state, pitch adjustment, and any playing loop will be cloned from one deck to another. This opens up a number of creative mixing possibilities, some of which we will explore below.
+
+.. image:: ../_static/deckclone.gif
+    :align: center
+
+How to use deck cloning
+-----------------------
+Deck cloning can be triggered several different ways:
+
+* Double press the :guilabel:`Load` button on your controller, mixer, or keyboard (this can be disabled in the Decks page of the preferences)
+* Invoke the :guilabel:`CloneFromDeck` control in a controller mapping (see the complete :ref:`Mixxx controls table <advanced-mixxxcontrols>`)
+* Drag and drop a track from one deck to another in the UI
+
+Any of these actions will result in the currently loaded track being loaded on the target deck at the current play position and pitch setting. If the track is playing on the source deck, it will play from the same exact position on the target deck.
+
+What you can do with deck cloning
+---------------------------------
+
+Playing with a single turntable
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In a typical DJ setup, you have two decks and a mixer, but with the deck clone feature, it's possible to play with a single deck. You can :term:`cue point <cue>`, beatmatch, and mix in the next track on a single deck. Once the track is mixed in, swap it to another channel using the deck clone feature, freeing up your single deck for the next track. This is useful if you are building out a DVS setup, but you only have one turntable so far or if you need to play with a minimal setup. For example, you may not have enough space for two decks or you might want to reduce the amount of gear you have to lug around. Finally, this method can be used as a backup way to keep playing if one of your turntables malfunctions.
```

Comment:
How about adding a link to https://www.youtube.com/watch?v=qLwsFQK4J6c ?

Reply:
This section could use more detail with step-by-step instructions.

Reply:
Alright then.  I will convert this and all the other topics that i will be adding into a step by step. Thank you!